decisions/version-compatibility.md

```diff
@@ -1,4 +1,4 @@
-# KDD-001: Versions and Compatibility
```

Comment:
@jmbrunskill is it important to identify KDD by something other then the title ?

Reply:
Probably not, I guess it serves a purpose similar to the 'code' in mSupply. A shorthand way to refer to a decision.
Feels right to have an id/code but I'd be fine if you wanted to remove it.